dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -893,11 +935,13 @@ private void updateInfo(Workflow oldWorkflow, Workflow newWorkflow) {
 
         }
 
-        if (workflowVersion.getDoiStatus() != Version.DOIStatus.CREATED) {
-            DOIGeneratorInterface generator = DOIGeneratorFactory.createDOIGenerator();
-            generator.createDOIForWorkflow(workflowId, workflowVersionId);
-            workflowVersion.setDoiStatus(Version.DOIStatus.REQUESTED);
-        }
+        List<Token> tokens = checkOnZenodoToken(user);
+        Token zenodoToken = Token.extractToken(tokens, TokenType.ZENODO_ORG);
+        final String zenodoAccessToken = zenodoToken == null ? null : zenodoToken.getContent();
```

Comment:
If the token is null, you shouldn't continue; you should throw a CustomWebApplicationException.

Reply:
For next sprint https://ucsc-cgl.atlassian.net/browse/SEAB-349